R/server.R

```diff
@@ -459,6 +459,47 @@ startApp <- function(appObj, port, host, quiet) {
     ),
     .globals$resourcePaths
   )
+
+  # throw an informative warning if a subdirectory of the
+  # app's www dir conflicts with another resource prefix
+  wwwDir <- httpuvApp$staticPaths[["/"]]$path
+  if (length(wwwDir)) {
+    wwwSubDirs <- list.dirs(wwwDir, recursive = FALSE, full.names = FALSE)
```

Comment:
It might make sense to use `recursive = TRUE` here, since a sub-sub-directory might conflict with a static path.